src/Tes.Runner.Test/Storage/TerraUrlTransformationStrategyTests.cs

```diff
@@ -53,11 +48,10 @@ private void SetupWsmClientWithAssumingSuccess()
                     Token = StubSasToken,
                     Url = $"{stubTerraBlobUrl}/{StubBlobName}?{StubSasToken}"
                 })
-                .Callback((Guid wksId, Guid resourceId, SasTokenApiParameters sasTokenParams,
-                    CancellationToken cancellationToken) =>
+                .Callback((Guid _, Guid _, SasTokenApiParameters sasTokenParams,
+                    CancellationToken _) =>
```

Comment:
For these two lines, sometimes the C# compiler recognizes arguments named `_` as "throwaway" and sometimes it doesn't, which can cause unexpected (and otherwise unexplained) future errors (usually having to do with being unable to cast values to [pick type]). Analyzers have all been upgraded to recognize the pattern "_n" where n is [0-9] (or possibly any series of digits?). So, I'd recommend replacing the three `_` with `_1`, `_2`, & `_3`, respectively.